frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,180 @@
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            ) : (
+              <tbody>
+                {users.map((user) => (
+                  <tr key={user.id}>
+                    <td>
+                      <Avatar
+                        avatarId={user.avatar_id}
+                        text={`${user.first_name ? user.first_name[0] : ''}${user.last_name ? user.last_name[0] : ''}`}
+                      />
+                      <span
+                        className="mx-3"
+                        style={{
+                          display: 'inline-flex',
+                          marginBottom: '7px',
+                        }}
+                        data-cy="user-name"
+                      >
+                        {user.name}
+                      </span>
+                    </td>
+                    <td className="text-muted">
+                      <a className="text-reset user-email" data-cy="user-email">
+                        {user.email}
+                      </a>
+                    </td>
+                    {user.status && (
+                      <td className="text-muted">
+                        <span
+                          className={`badge bg-${
+                            user.status === 'invited' ? 'warning' : user.status === 'archived' ? 'danger' : 'success'
+                          } me-1 m-1`}
+                          data-cy="status-badge"
+                        ></span>
+                        <small className="user-status" data-cy="user-status">
+                          {user.status}
+                        </small>
+                        {user.status === 'invited' && 'invitation_token' in user ? (
+                          <CopyToClipboard text={generateInvitationURL(user)} onCopy={invitationLinkCopyHandler}>
+                            <img
+                              data-tip="Copy invitation link"
+                              className="svg-icon"
+                              src="assets/images/icons/copy.svg"
+                              width="15"
+                              height="15"
+                              style={{
+                                cursor: 'pointer',
+                              }}
+                              data-cy="copy-invitation-link"
+                            ></img>
+                          </CopyToClipboard>
+                        ) : (
+                          ''
+                        )}
+                      </td>
+                    )}
+                    <td>
+                      <button
+                        type="button"
+                        style={{ minWidth: '100px' }}
+                        className={`btn btn-sm btn-outline-${user.status === 'archived' ? 'success' : 'danger'} ${
+                          unarchivingUser === user.id || archivingUser === user.id ? 'btn-loading' : ''
+                        }`}
```

Comment:
Let's simplify this using cx package
```suggestion
      import cx from 'classnames'
      ...
      className={cx('btn btn-sm', { 
        'btn-outline-danger': user.status === 'archived', 
        'btn-outline-success': user.status === 'active', 
        'btn-loading': unarchivingUser === user.id || archivingUser === user.id
     })}
```